sdk/lib/opentelemetry/sdk/trace/export/batch_span_processor.rb

```diff
@@ -47,7 +47,8 @@ def initialize(exporter:,
                          exporter_timeout_millis: Float(ENV.fetch('OTEL_BSP_EXPORT_TIMEOUT_MILLIS', 30_000)),
                          schedule_delay_millis: Float(ENV.fetch('OTEL_BSP_SCHEDULE_DELAY_MILLIS', 5_000)),
                          max_queue_size: Integer(ENV.fetch('OTEL_BSP_MAX_QUEUE_SIZE', 2048)),
-                         max_export_batch_size: Integer(ENV.fetch('OTEL_BSP_MAX_EXPORT_BATCH_SIZE', 512)))
+                         max_export_batch_size: Integer(ENV.fetch('OTEL_BSP_MAX_EXPORT_BATCH_SIZE', 512)),
+                         start_thread_on_boot: String(ENV['OTEL_BSP_START_THREAD_ON_BOOT']) !~ /false/i)
```

Comment:
```suggestion
                         start_thread_on_boot: String(ENV['OTEL_RUBY_BSP_START_THREAD_ON_BOOT']) !~ /false/i)
```
This is quite specific to Ruby, so we should adhere to the [Language Specific Environment Variables](https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/sdk-environment-variables.md#language-specific-environment-variables) recommendations in the spec.

Reply:
I will do as you request, however I don't think this is specific to Ruby. Any BSP implementation will need to have a worker thread, whether Python, Java, etc. and it will be useful to control whether such thread is eager or lazy-started.